src/components/Work/Work.tsx

```diff
@@ -200,6 +205,16 @@ const DoiPresentation: React.FunctionComponent<Props> = ({ doi }) => {
       <WorkMetadata metadata={doi} linkToExternal={true}></WorkMetadata>
       {exportMetadata()}
       {formattedCitation()}
+      {showFunding && (
+        <h3 className="member-results" id="work-funding">Funding</h3>
+      )}
+      {showFunding && (
```

Comment:
Minor style comment, no need to do check twice can just do it once for the whole thing right?

Reply:
I wasn't sure how to nest the code (first the condition then the loop) properly. Can look into it again.